ercc/Makefile

```diff
@@ -6,32 +6,23 @@
 TOP = ..
 include $(TOP)/build.mk
 
-ATTESTATION_DIR = ./attestation/
-ATTESTATION_PROVIDER = $(ATTESTATION_DIR)/provider.go
+build: ercc
 
-build: attestation ercc
-
-$(ATTESTATION_PROVIDER):
-	$(MAKE) -C $(ATTESTATION_DIR)
-
-attestation-clean:
-	$(MAKE) -C $(ATTESTATION_DIR) clean
-
-ercc: $(ATTESTATION_PROVIDER)
+ercc:
 	# ERCC's binary is created here.
 	# The binary is then referenced by the fpc-peer cli, inserted in the ERCC package,
 	# and eventually run by the external launcher.
-	$(GO) build $(GOTAGS)
+	$(GO) build $(GOTAGS) -o ercc main.go
 
 test: build
 	$(GO) test $(GOTAGS) -v ./...
 
-clean: docker-clean attestation-clean
+clean:
 	$(GO) clean
+	rm -f ercc
 
-ERCC_NAME?=ercc
-DOCKER_IMAGE?=$$(docker images | grep -- -$(ERCC_NAME)- | awk '{print $$1;}')
+generate:
+	$(GO) generate ./...
 
-docker-clean:
-	-if [ ! -z "$(DOCKER_IMAGE)" ]; then docker rmi -f $(DOCKER_IMAGE); fi
-	-if [ ! -z "$(DOCKER_IMAGE)" ]; then echo "\033[0;33mWARNING: ercc not deleted (peer still up?)\033[0m"; fi
+docker:
```

Comment:
oh this docker-clean cleans ercc images produced by a peer. Since we use either chaincode as a service or chaincode started using external builder this special clean command is not needed anymore. But I think a docker-clean is still helpful here. Let me bring back a docker clean.